src/server/api/controllers/qr.controller.js

```diff
@@ -0,0 +1,24 @@
+const knex = require('../../config/db');
+const moment = require('moment-timezone');
+
+const get = async (game) => {
+  try {
+    const now = moment().format();
+    const instance = await knex('game_instances').insert({
+      score: 0,
+      created_at: now,
+      updated_at: now,
+      fk_game_id: game,
```

Comment:
You have an error on Swagger:
`"insert into `game_instances` (`created_at`, `fk_game_id`, `score`, `updated_at`) values ('2020-06-20T21:03:13+02:00', 'this', 0, '2020-06-20T21:03:13+02:00') - Unknown column 'fk_game_id' in 'field list'"`
There is no 'fk_game_id' column in the table you are trying to perform an insert statement.

Reply:
Hey Ricardo if you look at diagram available here [https://dbdiagram.io/d/5ee687a19ea313663b3a885c](url) you see there is 'fk_game_id' in that table